src/footprintmonitoring/azext_footprintmonitoring/generated/_help.py

```diff
@@ -0,0 +1,238 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['footprintmonitoring profile'] = """
```

Comment:
footprintmonitoring is a quite long word for a cmdlet. Is possible to shorten it?

Reply:
renamed to "footprint"